src/osd/scheduler/mClockScheduler.h

```diff
@@ -33,8 +33,8 @@
 
 namespace ceph::osd::scheduler {
 
-constexpr uint64_t default_min = 1;
-constexpr uint64_t default_max = 999999;
+constexpr double default_min = 1.0;
+constexpr double default_max = 999999.0;
```

Comment:
Do we expect them to ever change at run-time? Shouldn't be typed as CONSTANTS?

Reply:
These are not expected to change at runtime.